net/knock/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem		1.0
+PortGroup		github 1.0
+
+github.setup		jvinet knock 258a27e5a47809f97c2b9f2751a88c2f94aae891
+version			0.7.8-20151227
+#name			knock
+categories		net
+platforms		darwin
+maintainers		nomaintainer
+license			GPL-2
+description		A port-knocking implementation
+long_description	${description}
+homepage		https://www.zeroflux.org/projects/knock
+#master_sites		http://www.zeroflux.org/proj/knock/files/
+checksums		rmd160	a95443251191b26f96fead78e8e25eceb29ae7f0 \
+			sha256	33640a796b338f07a28de7e31578c06bfabea52256ce14933776680a3074f85a \
+			size	31054
+
+depends_lib		port:libpcap
```

Comment:
While it is not strictly necessary, please change `depends_lib` to `depends_lib-append` because it is best practice when using portgroups to not overwrite any dependencies they might add.